aws/cloudformation/components/codeprojects.yml.erb

```diff
@@ -0,0 +1,73 @@
+<%# Note this file must be indented one level, as it is a child of 'Resources'. %>
```

Comment:
Can we use the `indent` method to help with this like we do [here](https://github.com/code-dot-org/code-dot-org/blob/179ecfbd02c7a6bb71666157cd966099fbaec2b1/aws/cloudformation/cloud_formation_stack.yml.erb#L469)?

Ideally, we would refactor `component` to take an indentation argument:
https://github.com/code-dot-org/code-dot-org/blob/7a10bfb359036ca0c7f662b30304d6b9e8e26da3/lib/cdo/cloud_formation/stack_template.rb#L76

Reply:
Components almost always represent sets of resources, which means they'd all be at '2' indent.

Using `indent`, we would have `indent(component('codeprojects'), 2)`

I feeli like I would need to keep my comment though, at least until there is consistency across all components. I do like the idea of reworking `component`, or maybe it's a new function that implies a list of resources? `resource_component` perhaps?

Reply:
Updated this again to use that.